staging/src/k8s.io/kubectl/pkg/cmd/proxy/proxy.go

```diff
@@ -185,6 +185,13 @@ func (o *ProxyOptions) Complete(f cmdutil.Factory) error {
 			RejectMethods: proxy.MakeRegexpArrayOrDie(o.rejectMethods),
 		}
 	}
+	if o.address == "0.0.0.0" {
```

Comment:
@aojea should this be something `net.ParseIP(o.address).IsUnspecified()` to catch ipv6 "listen to all" addresses as well?

Reply:
what Jordan says https://pkg.go.dev/net#IP.IsUnspecified

The reality is that the address is "all zeros", `0.0.0.0` or `::` are the representations , you have to use net.ParseIPSloppy() to avoid the linter problems